.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+ci:
+  autofix_prs: false
```

Comment:
its potentially helpful to leave this, eases the life for drive by contributors that just messed up syntax

Reply:
I would leave it set to `false` as contributors might not realize that someone/something else pushed new commits to their branch. I would expect contributors to have pre-commit installed locally, therefore they should run the checks before pushing the commit - either as an actual git hook or manually by `pre-commit run`.

There is still the possibility to fix it by adding a `pre-commit.ci autofix` comment to the PR. See https://pre-commit.ci/#configuration-autofix_prs